internal/transport/transport.go

```diff
@@ -698,7 +712,7 @@ type ClientTransport interface {
 // Write methods for a given Stream will be called serially.
 type ServerTransport interface {
 	// HandleStreams receives incoming streams using the given handler.
```

Comment:
"the Context given is used as the base context for all streams started on this transport."?

Reply:
As per comment above, will wait until further discussion on the server handler transport to add this docstring.

Reply:
Decided to do this, added this comment.